From b3292de1cdb0917c984f2a67ff97bf2d64ab2a22 Mon Sep 17 00:00:00 2001
From: Robert Mustacchi <rm@joyent.com>
Date: Fri, 3 Mar 2017 16:17:49 +0000
Subject: [PATCH] OS-5996 ctfmerge fails when using -l Reviewed by: Trent Mick
 <trent.mick@joyent.com> Reviewed by: Patrick Mooney
 <patrick.mooney@joyent.com> Approved by: Ryan Zezeski <rpz@joyent.com>

---
 usr/src/cmd/ctfmerge/ctfmerge.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/usr/src/cmd/ctfmerge/ctfmerge.c b/usr/src/cmd/ctfmerge/ctfmerge.c
index 5dea32b3aa..a05d6aab9b 100644
--- a/usr/src/cmd/ctfmerge/ctfmerge.c
+++ b/usr/src/cmd/ctfmerge/ctfmerge.c
@@ -10,7 +10,7 @@
  */
 
 /*
- * Copyright (c) 2015, Joyent, Inc.
+ * Copyright (c) 2017, Joyent, Inc.
  */
 
 /*
@@ -355,7 +355,7 @@ main(int argc, char *argv[])
 	 * We support a subset of the old CTF merge flags, mostly for
 	 * compatability.
 	 */
-	while ((c = getopt(argc, argv, ":d:fgj:L:o:t")) != -1) {
+	while ((c = getopt(argc, argv, ":d:fgj:l:L:o:t")) != -1) {
 		switch (c) {
 		case 'd':
 			g_unique = optarg;
-- 
2.21.0

